src/lib.rs

```diff
@@ -15,3 +30,45 @@ pub mod net;
 pub mod config;
 pub mod client;
 pub mod admin;
+
+#[cfg(windows)]
+pub mod windows {
```

Comment:
@senior @briancain Alright I think I have the appropriate code now lifted from https://github.com/rust-lang-nursery/rustup.rs/blob/master/src/rustup-utils/src/raw.rs#L821and I got the GUID from the Microsoft website.